src/queryWizard/QueryWizard.js

```diff
@@ -0,0 +1,54 @@
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import React from 'react'
+import { observer } from 'mobx-react'
+import DefaultCheckButton from '../layout/CheckButton'
+import DefaultModal from '../layout/Modal'
+import DefaultIcon from '../DefaultIcon'
+import WizardStep from './WizardStep'
+import { withStateLens } from '../utils/mobx-react-utils'
+
+export default withStateLens({ current: 0 })(
```

Comment:
Just use react state instead of `withStateLens`. You won't need observable either since it doesn't seem like you're observing anything worth a re-render.